### PR TITLE
Specify default value for task.concurrency in docs for FTE mode

### PR DESCRIPTION
**Pull request by posulliv**
## Release notes

( X) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
## Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Comment by mosabua** [comment]
Really ... what makes you say that it defaults to 8 in FTE mode? Can you confirm that is hardcoded like that @losipiuk ?

File: docs/src/main/sphinx/admin/properties-task.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 - **Type:** {ref}`prop-type-integer`
 - **Restrictions:** Must be a power of two
-- **Default value:** The number of physical CPUs of the node, with a minimum value of 2 and a maximum of 32
+- **Default value:** The number of physical CPUs of the node, with a minimum value of 2 and a maximum of 32. Defaults to 8 in fault tolerant execution mode
```
Comment (mosabua) [inline docs/src/main/sphinx/admin/properties-task.md:7]: If this is true we should - wrap this, link to FTE docs, and add end of sentence .

Comment (losipiuk) [reply docs/src/main/sphinx/admin/properties-task.md:7]: True:
https://github.com/trinodb/trino/blob/4db4f606e770a15e62063e9f7c0383ee657c6cca/core/trino-main/src/main/java/io/trino/server/ServerMainModule.java#L332-L334

Comment (mosabua) [reply docs/src/main/sphinx/admin/properties-task.md:7]: Hm .. can you check if this affects other properties as well @posulliv and just update them all at once and maybe even add a section in the FTE docs page for it.

Comment (posulliv) [reply docs/src/main/sphinx/admin/properties-task.md]: @mosabua the only property that changes its default in FTE mode is `exchange.compression-codec`. This one is already covered in the exchange properties docs.

**Review by mosabua — APPROVED** [review]
Thx

**Pull request merged**